chapters/modules/chapter_8/pages/solidity_verifier.adoc

```diff
@@ -0,0 +1,37 @@
+[id="verifiers_solidity"]
+
+= Verifiers in Solidity
+
+L1 smart contract that verifies STARK proofs from SHARP.
+
+== Example of a Verifier transaction
+Let's take a look at a transaction that was sent to the Verifier contract on Ethereum mainnet. 
+https://etherscan.io/tx/0xde0bcb45698e8ef67fef8bff8615715dac0cf0882a131a21273e58607330d5f9[0xde0bcb45698e8ef67fef8bff8615715dac0cf0882a131a21273e58607330d5f9]
+
+=== Transaction trace
+- Verifier contract is a proxy contract so when it's called, implementation contract is called instead. Implementation contract is GpsStatementVerifier contract https://etherscan.io/address/0x6cb3ee90c50a38a0e4662bb7e7e6e40b91361bf6[0x6cB3EE90C50a38A0e4662bB7e7E6e40B91361BF6]
```

Comment:
it would be beneficial to clarify how we can ascertain that the implementation contract was indeed invoked. Additionally, could you elucidate the linkage mechanism between the Verifier contract and the GpsStatementVerifier contract?

Reply:
- I'm not sure how should we `show` that implementation contract was invoked?
- for the linkage, do you think showing this upgrade [txn](https://etherscan.io/tx/0xbaa8ffb1b7e5177dbf75de753b9b2ff2fc313b244ff910cfd6d7f1f6254b6e1a) is enough? That txns is from [Dune query](https://dune.com/queries/2486605)

Reply:
* Looks good as is! Thought about if we could clarify things a bit more, but honestly, what we have now is already solid.
* Yes, I think that'll do the trick.